lib/ContractInteract/BrandedToken.js

```diff
@@ -411,6 +415,61 @@ class BrandedToken {
       this.contract.methods.rejectStakeRequest(stakeRequestHash),
     );
   }
+
+  /**
+   * This reedems brandedToken, must be called by beneficiary.
+   *
+   * @param {string} btToRedeem Amount of brandedTokens to redeem.
```

Comment:
The variable can also be called `amount`; and in comments specify it is in `attoBrandToken` (10^-18 BT)

Reply:
Done!
Update with below: 
"Amount of brandedTokens to redeem. Amount unit is wei i.e.1 BT = 10^18 wei."